packages/test/test-service-load/testConfig.json

```diff
@@ -144,6 +144,26 @@
                 "min": 900000,
                 "max": 1800000
             }
+        },
+        "custom_profile_6":  {
```

Comment:
probably could use a better name? perhaps `unstaggeredWrite`? (also I notice there's inconsistent naming convention between the different profiles, we should probably clean that up)

Reply:
Done!